packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 42.5.0 - March 1, 2018
+### Added
+#### salesforcedx-vscode-apex
+
+* Find All References for user-defined classes, enums, and interfaces (from type declarations and type usages) ([PR #324](https://github.com/forcedotcom/salesforcedx-vscode/pull/324))
+* Find matching references for Apex methods ([PR #324](https://github.com/forcedotcom/salesforcedx-vscode/pull/324))
```

Comment:
What does this mean?

Reply:
Can we fold this in to the one on top since it references the same PR?

Find All References for user-defined classes, enums, interfaces, and methods (from type declarations and type usages) ([PR #324](https://github.com/forcedotcom/salesforcedx-vscode/pull/324))

Reply:
No objections, let me make the change